app/main.py

```diff
@@ -1,8 +1,26 @@
 class Person:
-    # write your code here
-    pass
+
+    people: [dict[str]] = {}
```

Comment:
dict accept two arguments. If this will not work just remove annotations
```suggestion
    people: dict[str, "People"] = {}
```

Reply:
> dict accept two arguments. If this will not work just remove annotations

Thank you for all the remarks. 
Regarding annotation: people: dict[str, "People"] = {}  --- did bot worked, annotation remowed.

Reply:
people: dict[str, "People"] ->  people: dict[str, "Person"]

Reply:
My apologies, but I am absolutely confused how to make annotation correctly without issues. 

1st change was: people: [dict[str]] = {} - all worked, tests passed, no flake8 errors
I've got updade from @ArsenPidhoretskyi  about 2 required arguments for such annotation. His alternative proposal was: people: dict[str, "People"] = {}
error: Unresolved reference "People" - or "F821 undefined name 'People'" of flake8 app
2d change: Due to above mention error I've done no annotation, as was instructed in case of it's unsuccess. 

3d change.  @olena-hul 's proposal: people: dict[str, "People"] -> people: dict[str, "Person"]
This annotation also not working and gives errors. Please see attached screnshoots.

 Advise me please how to make correct annotation without breaking the work of my code in this task. 


![image](https://user-images.githubusercontent.com/121687558/215289344-9a9799c8-620f-4dd0-abec-dcbf2c809ccb.png)
![image](https://user-images.githubusercontent.com/121687558/215289373-6f55e6b5-c20f-4356-b0f1-abd25f44c260.png)

Reply:
I've chose simplest: people: dict = {}
:-)